radio_buttons/radio_buttons.less

```diff
@@ -0,0 +1,62 @@
+@import '_vars.less';
+html {
+  font-size: 10px;
+}
+
+.radio-buttons {
+
+  input[type=checkbox] + label:hover:before,
+  input[type=radio] + label:hover:before {
+     background: @blue;
+  }
+  [type="radio"]:checked,
+  [type="radio"]:not(:checked) {
+      position: absolute;
+      left: -9999em;
```

Comment:
This is something that would benefit from the mixin I was referring to in #9. It would also standardize the way we move elements out of sight.